shaka/src/memory/object_tracker.h

```diff
@@ -93,7 +93,7 @@ class ObjectTracker final : public PseudoSingleton<ObjectTracker> {
   void UnregisterAllObjects();
 
   /** @return Whether the given object is alive in JavaScript. */
-  bool IsJsAlive(Traceable* object) const;
+  bool IsJsAlive(Traceable* object, uint64_t now) const;
```

Comment:
You changed the header, but not the implementation.

Reply:
Ack, sorry. I'll see about getting a build environment set up.